mscore/inspector/inspector_ambitus.ui

```diff
@@ -910,21 +894,5 @@
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>hasLine</sender>
-   <signal>toggled(bool)</signal>
-   <receiver>resetLineWidth</receiver>
-   <slot>setEnabled(bool)</slot>
```

Comment:
Aren't these connections needed? Or are they handled elsewhere?

Reply:
I believe those were just mistakes, are not needed at all and never were.

Reply:
Also these were duplicates

Reply:
As written in the commit message, I'm not sure any of those Signals/Slots are needed (actually I'm pretty sure none is). But I removed only those I got warnings about.

Reply:
My guess is that those stem from times before they got changed into inspector dialog?